racket/src/cs/rumble/system.ss

```diff
@@ -74,6 +78,10 @@
 
 (define arch-symbol
   (case (reflect-machine-type)
+    [(a6ios ta6ios)
+     'x86_64]
+    [(arm64ios tarm64ios)
+     'aarch64]
```

Comment:
I would mildly prefer adding these to the existing clauses that return `'x86_64` and `'aarch64`, respectively.

As I wrote in https://github.com/racket/racket/pull/4702 and https://github.com/racket/racket/issues/3948#issuecomment-1481803208, I think a much better solution, though out of scope for this PR, would be to push more of this work to Chez Scheme. I'm still not sure of a design I'd find fully convincing for that, though `(constant architecture)` might be enough for this specifically &hellip;